app/notification.go

```diff
@@ -422,8 +423,7 @@ func (a *App) SendNotifications(post *model.Post, team *model.Team, channel *mod
 	if *a.Config().ServiceSettings.CollapsedThreads != model.COLLAPSED_THREADS_DISABLED && post.RootId != "" {
 		thread, err := a.Srv().Store.Thread().Get(post.RootId)
 		if err != nil {
-			mlog.Error("Cannot get thread", mlog.String("id", post.RootId))
-			return nil, err
+			return nil, fmt.Errorf("cannot get thread %q, %w", post.RootId, err)
```

Comment:
We usually don't use the `%w` convention in the main repo, but just stick to `errors.Wrap(f)`